Home.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+N_Second = input()
```

Comment:
Непонятно, какой информации ты ожидаешь от пользователя. Следует явно указать, что пользователю нужно ввести, например, так:
 ```suggestion
N_Second = input("Введите N – количество секунд для перевода")
```